src/snowflake/snowpark/functions.py

```diff
@@ -5356,32 +5356,33 @@ def array_remove(array: ColumnOrName, element: ColumnOrLiteral) -> Column:
         -------------
         <BLANKLINE>
 
-        >>> df.select(array_remove(df.data, lit('2').cast(VariantType())).alias("objects")).show()
+        >>> df.select(array_remove(df.data, None).alias("objects")).show()
```

Comment:
It is hard to judge if the changes here can be a behavior change. Can you ensure all the changes you are adding in examples and tests are new. Not modifying old tests so we know old behavior is the same

Reply:
Thanks for your review. I have ensured that the original tests remain intact and have added new tests to cover the changes introduced, ensuring that both the old and new behavior are properly validated.